Gemfile

```diff
@@ -46,6 +46,15 @@ group :development, :test do
   gem "parallel_tests"
 end
 
+group :test do
+  # Adds support for Capybara system testing and selenium driver
+  gem 'capybara'
+  gem 'selenium-webdriver'
+  # Easy installation and use of chromedriver to run system tests with Chrome
+  gem 'chromedriver-helper'
+  gem 'a15k_interactions_api', path: './clients/0.1.0/ruby'
```

Comment:
How do we later also test against 0.2.0?

Reply:
we can't :(

I think we might be able to do that with doing some LOAD_PATH trickery.  We'd have to add and remove the gem path in a before/after rspec block, but I've left that exploration until we actually have a version 2.